index.html

```diff
@@ -171,5 +175,76 @@ <h2>Multi-Post Stories</h2>
         </div>
       </div>
     </section>
+    <section class="contact-form">
+      <div class="description">
+        <h1>Contact Us</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form action="https://formspree.io/f/mzbodazz">
+        <label for="name"></label>
+        <input
+          required
+          type="text"
+          placeholder="Your name"
+          name="name"
+          id="name"
+          maxlength="30"
+        />
+
+        <label for="mail"></label>
+        <input
+          required
+          type="email"
+          placeholder="jeremy@gmail.com"
+          name="email"
+          id="mail"
+        />
+        <label for="message"></label>
+        <textarea
+          required
+          name="message"
+          id="message"
+          placeholder="Enter text here"
+          maxlength="500"
+          
+
+        ></textarea>
+        <div id="contactButton">
+        <button type="submit">Submit</button>
```

Comment:
- [ ] On requirement states that `Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.` Kindly follow this requirement by adding the button interactions. here is the corresponding design 👇  
![image](https://user-images.githubusercontent.com/65068771/150122583-3cb51251-57aa-45ce-ba48-c23121b6e596.png)